examples/pytorch/graphsage/experimental/README.md

```diff
@@ -28,15 +28,31 @@ will be able to access the partitioned data.
 
 We need to run a server on each machine. Before running the servers, we need to update `ip_config.txt` with the right IP addresses.
 
+On each of the machines, set the following environment variables.
+
+```bash
+export DGL_ROLE=server
```

Comment:
We set `DGL_ROLE=serve`r here. When to set `DGL_ROLE=client`?

Reply:
the default behavior is working as a client. we don't need to set it to be client. we could ask users to set it to client, but it doesn't matter.